scripts/verify.py

```diff
@@ -47,12 +47,17 @@ def main(checks):
             run("./scripts/check_requirements_and_setup.py", shell=True, check=True)
             print("check requirements passed")
 
+        if "check-files-sizes" in checks:
+            print("Checking all added files have size <= 2MB", flush=True)
+            run("./scripts/check_files_sizes.sh", shell=True, check=True)
```

Comment:
Do you need to pass a specific size to the script?

Reply:
oh yea